packages/chrome-extension/src/utils/storage.ts

```diff
@@ -0,0 +1,47 @@
+import browser from 'webextension-polyfill';
+
+type StorageCacheOptions = {
+  storageArea?: 'local' | 'sync';
```

Comment:
It's not a requirement, no, however it's in there due to a comment in the original methods. In theory, this would limit it to local/sync, as there are other options as well.

```ts
// Use chrome.storage (local or sync) to persist Clerk client JWT.
// More information athttps://developer.chrome.com/docs/extensions/reference/storage
```